addons/subscription/models/subscription.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# TODO:
+#   Error treatment: exception, request, ... -> send request to user_id
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+class SubscriptionDocument(models.Model):
+    _name = "subscription.document"
+    _description = "Subscription Document"
+
+    name = fields.Char(required=True)
+    active = fields.Boolean(help="If the active field is set to False, it will allow you to hide the subscription document without removing it.", default=True)
+    model_id = fields.Many2one('ir.model', string="Object", required=True)
```

Comment:
can you make seperate commit to change field name ? . also do not forgot to add oldname attribute.